src/app/client/src/index.html

```diff
@@ -26,6 +26,7 @@
   <input type="hidden" id="cdnUrl" value="<%=cdnUrl%>" />
   <input type="hidden" id="theme" value="<%=theme%>" />
   <input type="hidden" id="defaultPortalLanguage" value="<%=defaultPortalLanguage%>" />
+  <input type="hidden" id="extContWhitelistedDomains" value="<%=extContWhitelistedDomains%>"/>
```

Comment:
where are we using this variable - "extContWhitelistedDomains"?

Reply:
It will be sent in window.config of generic editor, file is added now